### PR TITLE
LPS-129342 Add consistent whitespace to properties

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//
+// Check GETTING_STARTED.markdown#liferay.workspace.target.platform.version
+// for information regarding usage.
+//
+
+/*subprojects {
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle]: Lets remove this comment and instead add this section of code to GETTING_STARTED.

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle [hunk 2]
```diff
@@ -0,0 +1,10 @@
+//
+// Check GETTING_STARTED.markdown#liferay.workspace.target.platform.version
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle]: So let's have a section in GETTING_STARTED for customzing the build with the root build.gradle. And in one section of the customizations possible we can have the BOM stuff.

Comment (gamerson) [reply modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle]: i.e. in this file i just want a link to the GETTING_STARTED, no commented out code (makes it hard to upgrade).

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle [hunk 3]
```diff
@@ -0,0 +1,3 @@
+//
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/build.gradle]: Lets make this just one single line.

File: modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/platform.bndrun [hunk 4]
```diff
@@ -1,15 +1,8 @@
...
-## SUCCESS: The succesful result is a list of all the artifacts needed to run
-## without any resolution errors.
-##
-## FAILURE: A failure will indicate missing a requirement. Correct the missing
-## requirement and rerun the task.
-##
+#set($h1 = '##')
+$h1
+$h1 Check GETTING_STARTED.markdown#platform.bndrun for 
+$h1 information regarding these values.
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/platform.bndrun]:
```suggestion
$h1 more information.
```

**Review by gamerson — COMMENTED** [review]
Once you've made these changes, just go ahead and send to Peter.  Thanks.

**Comment by lawrence-lee** [comment]
submitted to Hugo https://github.com/hhuijser/liferay-portal/pull/5381

**Pull request closed without merging**